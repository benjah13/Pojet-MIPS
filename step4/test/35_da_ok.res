
	Disassembly of section .text

00000000
      00: 00000000	NOP
      04: 00000000	NOP
