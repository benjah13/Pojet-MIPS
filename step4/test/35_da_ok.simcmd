# test da ok 

da 0x000:2
