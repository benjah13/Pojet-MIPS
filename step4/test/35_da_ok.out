00000000:	NOP
00000004:	NOP
